djmoney/models/fields.py

```diff
@@ -141,6 +141,42 @@ def set_currency(self, obj, value):
             setattr(obj, self.currency_field_name, value)
 
 
+class LinkedCurrencyMoneyFieldProxy(MoneyFieldProxy):
+    """
+    Overridden to support fetching currency across relationships.
+    """
+
+    def __init__(self, field, currency_field_name=None):
+        super().__init__(field)
```

Comment:
We still support Python 2.7, where this syntax is invalid

Reply:
Ah gotchya, ok, lemme fix that (there are a few other 3.x only lines of code too)

Reply:
@Stranger6667 2.7 compat fixes pushed